packages/web/pages/smart-accounts/index.tsx

```diff
@@ -0,0 +1,424 @@
+import { fromBase64, toBase64 } from "@cosmjs/encoding";
+import { PrivKeySecp256k1 } from "@keplr-wallet/crypto";
+import { isNil } from "@osmosis-labs/utils";
+import { useMutation } from "@tanstack/react-query";
+import { observer } from "mobx-react-lite";
+import type { NextPage } from "next";
+import { FunctionComponent } from "react";
+
+import { Button } from "~/components/buttons";
+import { OsmoverseCard } from "~/components/cards/osmoverse-card";
+import { CheckBox } from "~/components/control";
+import Spinner from "~/components/spinner";
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+import { Authenticator } from "~/server/queries/osmosis/authenticators";
```

Comment:
Would be good to import this as a type. Otherwise, I believe it would import the server code into FE IIRC.

`import type { Authenticator } from "~/server/queries/osmosis/authenticators";`